Deploy/configparser.cpp

```diff
@@ -693,11 +696,12 @@ bool ConfigParser::parseDeployMode() {
 
         if (DeployCore::isSnap()) {
             QuasarAppUtils::Params::log("If you are using qmake from the system repository,"
-                                        " then you should use the classic version of the CQtDeployer instead of the snap version."
-                                        " This is due to the fact that the snap version runs in an isolated container and has limited access"
-                                        " to system utilities and the environment. "
+                                        " then you must use the classic version of the CQtDeployer instead of the snap version."
+                                        " This is due to the fact that the snap version"
+                                        " runs in an isolated container and has limited access"
+                                        " to system utilities and an environment. "
```

Comment:
Why do you remove the 'the' and 'an' articles ?
both suggested options (mine and yours) are syntactically correct

Reply:
@pzhlkj6612

Reply:
1. "CQtDeployer" is a proper noun, so no article is needed.
2. For "environment", sorry for my mistake, I think using "the" is better because it refers to a generic thing - the environment of a nonspecific operating system.

```suggestion
                                        " then you must use the classic version of CQtDeployer instead of the snap version."
                                        " This is due to the fact that the snap version"
                                        " runs in an isolated container and has limited access"
                                        " to system utilities and the environment. "
```

Reply:
i agree